### ISSUE TITLE
Build issue in Julia 1.6.0-rc1

### ISSUE THREAD

**Issue by Mattriks** [JuliaData/SplitApplyCombine.jl#36]
Julia 1.6.0-rc1, in a fresh env (on a windows pc):
```julia
(@gsbase) pkg> add SplitApplyCombine
    Updating registry at `~\.julia\registries\General`
    Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Resolving package versions...
    Updating `~\.julia\environments\gsbase\Project.toml`
  [03a91e81] + SplitApplyCombine v1.1.4
    Updating `~\.julia\environments\gsbase\Manifest.toml`
  [85a47980] + Dictionaries v0.3.7
  [313cdc1a] + Indexing v1.1.0
  [03a91e81] + SplitApplyCombine v1.1.4
  [2a0f44e3] + Base64
  [b77e0a4c] + InteractiveUtils
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [9a3f8284] + Random
  [9e88b42a] + Serialization
  [8dfed614] + Test
0 dependencies successfully precompiled in 1 seconds (, 2 skipped during auto due to previous errors)
1 dependency errored

julia> using SplitApplyCombine
[ Info: Precompiling SplitApplyCombine [03a91e81-4c3e-53e1-a0a4-9c0c8f19dd66]
ERROR: LoadError: ArgumentError: Package Indexing [313cdc1a-70c2-5d6a-ae34-0150d3930a38]
 is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.
```
instantiate doesn't help here because Indexing.jl doesn't have a Project.toml.

**Comment by juliohm** [comment]
And this issue is affecting the installation of the GeoStats.jl stack as well since we moved from DataFrames.jl to TypedTables.jl

cc: @quinnj

**Comment by andyferris** [comment]
We released Indexing v1.1.1 - does this fix your problem?

**Comment by Mattriks** [comment]
Yes thank you!
